### DISCUSSION TITLE
docker compose部署运行后，小火箭获取订阅节点失败

### DISCUSSION THREAD

**Discussion by DuckDuck88** [vvbbnn00/WARP-Clash-API#120] [Q&A]
如图，加入了public key，获取节点时一直提醒无法获取
![88f554c75a14563819e68f802bd76bf](https://github.com/vvbbnn00/WARP-Clash-API/assets/45893278/b3218883-ffd9-468f-b383-fde3299511fe)

**Comment by rexyan** [comment]
一样

**Comment by vvbbnn00** [comment]
你好，请提供复现bug所需的具体步骤，例如：

1. 执行命令 `docker-compose up -d`
2. 点击按钮 `Button1`
3. 访问URL `http://xxxx`

如有额外信息（如运行日志、浏览器的响应数据等）可助于排查问题，请在此处添加。注意保护个人隐私，不要公开敏感信息（包括但不限于：IP地址、LicenseKey、PrivateKey、SECRET_KEY）。

**Comment by 1447262390** [comment]
openclash也一样
![image](https://github.com/vvbbnn00/WARP-Clash-API/assets/51668733/3b486c1f-2e09-4552-92b5-e8911bf237b6)

**Comment by 1447262390** [comment]
我补充一下，当我尝试使用Surge放在openclash的时候，发现可以，Clash、ShadowRocket这两个就不可以

**Comment by 1447262390** [comment]
我发现，即使订阅其它的，比如clash，订阅失败内核日志如下
![image](https://github.com/vvbbnn00/WARP-Clash-API/assets/51668733/84f55b36-94cc-4f56-bfc9-d8d809d7aebf)

**Comment by vvbbnn00** [comment]
> 我发现，即使订阅其它的，比如clash，订阅失败内核日志如下 ![image](https://private-user-images.githubusercontent.com/51668733/308874491-84f55b36-94cc-4f56-bfc9-d8d809d7aebf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkyMDE0NzUsIm5iZiI6MTcwOTIwMTE3NSwicGF0aCI6Ii81MTY2ODczMy8zMDg4NzQ0OTEtODRmNTViMzYtOTRjYy00ZjU2LWJmYzktZDhkODA5ZDdhZWJmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI5VDEwMDYxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBjODljMmNmNGZhYzExNTk2ZjU1MGFjNzkyNTQ0ZTkyNDViMWMxY2QxZDA4OGJhNTYyMzI1NjYzNDJiNzc0ZjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9Zs5JT2_NHyOSd-0slhvzsHmUG-ODWgOFV4sdAifFss)

这个应该是内核版本的问题：https://github.com/vvbbnn00/WARP-Clash-API/issues/19

**Comment by 1447262390** [comment]
感谢回复，我参考你的建议，把内核切换成meta后成功解决问题，非常感谢开源并维护这么好的工具

**Comment by haowen2000** [comment]
我的小火箭也不能订阅

**Reply by haowen2000** [reply]
和一楼的一样，不能获取订阅

**Reply by haowen2000** [reply]
升级最新的小火箭就可以了
